source/intro_rl/offline_rl_en.rst

```diff
@@ -0,0 +1,81 @@
+- Behavior policy (behavior policy is the policy that the agent uses to collect dataset through interacting with the environment)
+
+
+In the end, we will list and answer some common questions raised in the domain of reinforcement learning for reference.
+
+Reinforcement Learning Preliminaries
+------------------------------------
+
+The goal of RL is to learn a policy that maximizes the expected episodic discounted reward in a infinite-horizon Markov Decision Process~(DMP) defined by :math:`(\mathcal{S},\mathcal{A}, \mathcal{R}, p, \gamma)`, with the state space :math:`\mathcal{S}`, the action space :math:`\mathcal{A}`, reward function :math:`\mathcal{R}`, transition dynamics :math:`p`, and discount factor :math:`\gamma \in \left(0, 1 \right)`. 
+Following a given policy :math:`\pi(\mathbf{a} \mid \mathbf{s})`, the objective of RL is formulated by :math:`\mathbb{E}_{\mathbf{a} \sim \pi(\mathbf{a} \mid \mathbf{s})} \sum_{t=0}^{\infty} \gamma^{t}r_{t+1}`, which is the expected cumulative sum of rewards when following the policy in the MDP, where the importance of the horizon is determined by the discount factor.
+Q-Learning methods maintain a Q-function :math:`Q^{\pi}(\mathbf{s}, \mathbf{a})=\mathbb{E}_{\pi}\left[\sum_{t=0}^{\infty} \gamma^{t} r_{t+1} \mid \mathbf{s}_{0}=\mathbf{s}, \mathbf{a}_{0}=\mathbf{a}\right]`, which measures the expected discounted return following a given policy :math:`\pi(\mathbf{a} \mid \mathbf{s})`.
+
+Offline Reinforcement Learning
+------------------------------------
+
+Offline RL has a assumption that the agent cannot interact with the environment before the evaluation. 
+In the Offline RL setting, the agent is trained in a collected dataset :math:`\mathcal{D}` which is sampled from the environment by a behavior policy :math:`\pi_{\beta}(\mathbf{a}\mid \mathbf{s})`. For actor-critic algorithm given a fixed dataset :math:`\mathcal{D} = \left\{ (\mathbf{s}, \mathbf{a}, r, \mathbf{s}^{\prime})\right\}`, we can represent value iteration and policy improvement by:
+
+.. math::
+
+   \hat{Q}^{k+1} \leftarrow \arg\min_{Q} \mathbb{E}_{\mathbf{s}, \mathbf{a} \sim \mathcal{D}} \left[ \left(\hat{\mathcal{B}}^\pi \hat{Q}(\mathbf{s}, \mathbf{a})  - Q(\mathbf{s}, \mathbf{a}) \right)^2 \right],
+   \\
+   \hat{\pi}^{k+1} \leftarrow \arg\max_{\pi} \mathbb{E}_{\mathbf{s} \sim \mathcal{D}, \mathbf{a} \sim \pi^{k}(\mathbf{a} \mid \mathbf{s})}\left[\hat{Q}^{k+1}(\mathbf{s}, \mathbf{a})\right],
+
+
+where the :math:`\hat{\mathcal{B}}^\pi` represents the bellman operator on the fixed dataset following the policy :math:`\hat{\pi} \left(\mathbf{a} \mid \mathbf{s}\right)`, :math:`\hat{\mathcal{B}}^\pi \hat{Q}\left(\mathbf{s}, \mathbf{a}\right) = \mathbb{E}_{\mathbf{s}, \mathbf{a}, \mathbf{s}^{\prime} \sim \mathcal{D}}[ r(\mathbf{s}, \mathbf{a})+\gamma \mathbb{E}_{\mathbf{a}^{\prime} \sim \hat{\pi}^{k}\left(\mathbf{a}^{\prime} \mid \mathbf{s}^{\prime}\right)}\left[\hat{Q}^{k}\left(\mathbf{s}^{\prime}, \mathbf{a}^{\prime}\right)\right] ]`
+
```

Comment:
recheck all these equations above. Some ``Q`` forgets its `hat` and some ``Q``s forget k as superscript